erfa/tests/test_erfa.py

```diff
@@ -19,6 +19,28 @@ def __init__(self, *args, **kwargs):
             pass
 
 
+def embedded_liberfa(path=erfa.ufunc.__file__):
+    import platform
+    import subprocess
+
+    lddcmd = ['ldd']
+    if platform.system() == 'Darwin':
+        lddcmd = ['otool', '-L']
+    elif platform.system() == 'Windows':
+        # lddcmd = ['']  # TODO
+        return None
+
+    lddcmd.append(path)
+    try:
+        out = subprocess.run(lddcmd, check=True,
+                             encoding='utf-8', stdout=subprocess.PIPE)
+                             # capture_output=True, text=True)  # Python 3.7
```

Comment:
Is this needed, or does the above work on all supported python?

Reply:
Yes, the current implementation should work always.